exchange/wrappers/buzzex.js

```diff
@@ -0,0 +1,303 @@
+const Buzzex = require('buzzex-gekko-npm');
+const moment = require('moment');
+const _ = require('lodash');
+const exchangeUtils = require('../exchangeUtils');
+const retry = exchangeUtils.retry;
+const scientificToDecimal = exchangeUtils.scientificToDecimal;
+
+const marketData = require('./buzzex-markets.json');
+
+const Trader = function(config) {
+  _.bindAll(this);
+
+  if(_.isObject(config)) {
+    this.key = config.key;
+    this.secret = config.secret;
+    this.currency = config.currency.toUpperCase()
+    this.asset = config.asset.toUpperCase();
+  }
+
+  this.name = 'Buzzex';
+  this.since = null;
+  
+  this.market = _.find(Trader.getCapabilities().markets, (market) => {
+    return market.pair[0] === this.currency && market.pair[1] === this.asset
+  });
+  this.pair = this.market.book;
+
+  this.interval = 3100;
+
+  this.buzzex = new Buzzex(
+    this.key,
+    this.secret,
+    {timeout: +moment.duration(60, 'seconds')}
+  );
+}
+
+const recoverableErrors = [
+  
+];
+
+// errors that might mean
+// the API call succeeded.
+const unknownResultErrors = [
+  'Response code 502',
+  'Response code 504',
+  'Response code 522',
+  'Response code 520',
+]
+
+const includes = (str, list) => {
+  if(!_.isString(str))
+    return false;
+
+  return _.some(list, item => str.includes(item));
+}
+
+Trader.prototype.handleResponse = function(funcName, callback, nonMutating, payload) {
+  return (error, body) => {
+
+    if(!error && !body) {
+      error = new Error('Empty response');
+    }
+
+    if(error) {
+      if(includes(error.message, recoverableErrors)) {
+        error.notFatal = true;
+      }
+
+      if(includes(error.message, ['Rate limit exceeded'])) {
+        error.notFatal = true;
+        error.backoffDelay = 2500;
```

Comment:
This is kraken specific due to their API being very slow, is this the case as well for Buzzex? Else 500ms would be a more sane default I think.